tools/ks3util/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM pingcap/pingcap-base:v1
+RUN set -eux; \
+	case `uname -m` in \
+    x86_64) \
+        url=https://ks3util-version-update.ks3-cn-beijing.ksyuncs.com/2.4.1/ks3util-linux-amd64; \
+        ;; \
+    aarch64) \
+        # todo update url after ksyun fix this
+        url=http://localhost:8000/ks3util-linux-arm64; \
+        ;; \
+    *) \
+        echo >&2 "error, not supported platform";exit 1; \
+        ;; \
+    esac; \
+    curl -o /bin/ks3util "$url"; \
+    chmod a+x /bin/ks3util
```

Comment:
please add it into other repo: artifacts.git

Reply:
this is a builder image, where should it be placed?

Reply:
dockerfiles/ci/ks3util

> this is a builder image, where should it be placed?